src/js/page/odoo/commands/common/renew_database.mjs

```diff
@@ -0,0 +1,46 @@
+// @flow strict
+// Copyright Baptiste <swano@ik.me>
+// License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+// $FlowIgnore
+import i18n from 'i18next';
+import { v4 as uuidv4 } from 'uuid';
+import rpcQuery from '@odoo/rpc';
+import type { CMDCallbackArgs, CMDCallbackContext, CMDDef } from '@trash/interpreter';
+import type Terminal from '@odoo/terminal';
+
+async function cmdRenewDatabase(this: Terminal, kwargs: CMDCallbackArgs, ctx: CMDCallbackContext) {
+  const uuid = uuidv4();
+  const now = new Date();
+  const createDate = now.toISOString().slice(0, 19).replace('T', ' ');
+  const expirationDate = new Date(now.setDate(now.getDate() + 30)).toISOString().slice(0, 19).replace('T', ' ');
+
+  const params = [
+    { param: 'database.uuid', value: uuid },
+    { param: 'database.create_date', value: createDate },
+    { param: 'database.expiration_date', value: expirationDate },
+    { param: 'database.expiration_reason', value: false },
+  ];
+
+  for (const { param, value } of params) {
+    await rpcQuery({
+      method: 'set_param',
+      model: 'ir.config_parameter',
+      args: [param, value],
+      kwargs: { context: await this.getContext() },
+    });
+  }
+
+  ctx.screen.print(i18n.t('cmdRenewDatabase.result.success', 'Database renewed successfully'));
+  return params;
+}
+
+export default function (): Partial<CMDDef> {
+  return {
+    definition: i18n.t('cmdRenewDatabase.definition', 'Renew the database'),
+    callback: cmdRenewDatabase,
+    detail: i18n.t('cmdRenewDatabase.detail', 'Re-generate a new uuid and expiration date for your development database.'),
+    args: [],
```

Comment:
It would be nice to give more options to configure each parameter separately. Maybe you just want to change the date and not renew the uuid. Make the command more flexible :)

Reply:
looks good to me... maybe better "renew-dbuuid"?